packages/runtime/test/app.test.js

```diff
@@ -330,6 +343,7 @@ test('logs errors if an env variable is missing', async (t) => {
   }, /exited/)
   assert.strictEqual(process.exit.mock.calls.length, 1)
   assert.strictEqual(process.exit.mock.calls[0].arguments[0], 1)
+  t.mock.reset()
```

Comment:
This will happen automatically, so unless the rest of the test needs these to be reset, you don't need to do this.